packages/jitm/src/Manager.php

```diff
@@ -53,7 +65,8 @@ private function __construct() {
 	 * @return string The Jetpack emblem
 	 */
 	function get_emblem() {
-		return '<div class="jp-emblem">' . Jetpack::get_jp_emblem() . '</div>';
+		$jetpack_logo = new Jetpack_Logo();
+		return '<div class="jp-emblem">' . $jetpack_logo->get_jp_emblem() . '</div>';
```

Comment:
Is it worth having a naming scheme for methods that render output so we can tell if it's a URL, html, or implements hooks that render something later?

e.g. `get_jp_emblem_html`

Reply:
Or echo directly to output, e.g. `render_jp_emblem_html`